packages/react-components/theme-designer/src/components/Content/Content.tsx

```diff
@@ -30,24 +29,19 @@ const useStyles = makeStyles({
 
 export const Content: React.FC<ContentProps> = props => {
   const styles = useStyles();
-  const [isDark, setIsDark] = React.useState<boolean>(false);
-
-  const toggleTheme = React.useCallback(() => setIsDark(!isDark), [isDark, setIsDark]);
-
-  const theme = isDark ? props.darkTheme : props.lightTheme;
 
   return (
-    <FluentProvider theme={theme}>
+    <FluentProvider theme={props.theme}>
       <Alert intent="warning" action={{ appearance: 'transparent' }}>
         This tool is still a work in progress - colors are still subject to adjustment.
       </Alert>
       <div className={mergeClasses(styles.root, props.className)}>
         <Palette brandColors={props.brand} />
-        <Demo theme={theme} />
+        <Demo theme={props.theme} />
         <Divider />
-        <AccessibilityChecker theme={theme} />
+        <AccessibilityChecker theme={props.theme} />
         <Divider />
-        <TokenBoxes theme={theme} isDark={isDark} toggleTheme={toggleTheme} />
+        <TokenBoxes theme={props.theme} />
       </div>
     </FluentProvider>
```

Comment:
_nit:_ You could destructure the `theme` from `props` so you don't have to type `props.` everywhere, and allows you to have a place to destructure other things if you need to, like `brand` and `className` (also helps with bundle size slightly since you're writing less code, although I don't know how important that is for the theme designer).

```tsx
  const styles = useStyles();
  const { brand, className, theme } = props;

  return (
    <FluentProvider theme={theme}>
      <Alert intent="warning" action={{ appearance: 'transparent' }}>
        This tool is still a work in progress - colors are still subject to adjustment.
      </Alert>
      <div className={mergeClasses(styles.root, className)}>
        <Palette brandColors={brand} />
        <Demo theme={theme} />
        <Divider />
        <AccessibilityChecker theme={theme} />
        <Divider />
        <TokenBoxes theme={theme} />
      </div>
    </FluentProvider>
```